subsys/fs/Kconfig.fatfs

```diff
@@ -230,6 +230,27 @@ config FS_FATFS_REENTRANT
 	  access for each volume. Will create a zephyr mutex object for each
 	  FatFs volume and a FatFs system mutex.
 
+config FS_FATFS_LBA64
+	bool "Support for 64-bit LBA"
+	depends on FS_FATFS_EXFAT
+	help
+	  This option enables support for 64-bit LBA, which also
+	  enables GPT support.
+
+config FS_MULTI_PARTITION
```

Comment:
Would it not be better to call this `FS_FATFS_GPT_PARTITIONS` ?

Reply:
It should work without GPT also, I'll change it to FS_FATFS_MULTI_PARTITION - ok?

Reply:
Yes, that would be ok.

Reply:
fixed